java/query/query-core/src/main/java/sleeper/query/core/tracker/QueryTrackerStore.java

```diff
@@ -18,14 +18,48 @@
 
 import java.util.List;
 
+/**
+ * Interface for tracking the status of a query.
```

Comment:
"Interface for" can be omitted, see the Oracle style guide under "omit the subject and simply state the object": https://www.oracle.com/uk/technical-resources/articles/java/javadoc-tool.html

This retrieves the status of queries. It's not just for individual queries.